### DISCUSSION TITLE
OKD 4.11 After MCP automatic update, apiserver updates pod and cannot start

### DISCUSSION THREAD

**Discussion by wangmingwang123** [okd-project/okd#1845] [General]
**Describe the bug**
1. I woke up this morning and found that our OKD cluster suddenly had an MCP automatic update component, and the pod for updating the material component could not start
2. API server component upgrade failed and was downgraded

![image](https://github.com/okd-project/okd/assets/73093158/c436f6fc-24c4-485f-9ade-f98ba89a702c)
![image](https://github.com/okd-project/okd/assets/73093158/b4a60ac2-56aa-47f1-87fe-e1a2a0e208b2)

**Version**
4.11.0-0.okd-2023-01-14-152430

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive.
